jest-puppeteer.config.js

```diff
@@ -1,6 +1,10 @@
 module.exports = {
-  port: parseInt(process.env.JEST_PORT, 10)
-  // server: {
-  //   command: `./node_modules/.bin/serve --listen ${process.env.JEST_PORT} --no-clipboard ./dist`,
+  port: parseInt(process.env.JEST_PORT, 10),
+  // the launch options are helpful for debugging purposes
+
+  // launch: {
+  //   headless: false,
+  //   devtools: true,
   // },
+  browserContext: "incognito"
```

Comment:
i think it's good for tests not to share context with other tests! it would be nice if we could give it its own server maybe to not share context with local dev but ... that's a change for another day